README.md

```diff
@@ -42,6 +42,14 @@ You can use the hosted checkout with the following URL format: `https://<your-or
 
 For example: `https://yourbrand.checkout.commercelayer.app/PrnYhoVeza?accessToken=eyJhbGciOiJIUzUxMiJ9`
 
+If you are using [Commerce Layer CLI](https://github.com/commercelayer/commercelayer-cli), there is a [checkout plugin](https://github.com/commercelayer/commercelayer-cli-plugin-checkout/blob/main/README.md) that can generate the URL from SKU and bundle codes, or from an order id:
```

Comment:
I'll also re-word this to:

> If you are using the [Commerce Layer CLI](https://github.com/commercelayer/commercelayer-cli), there is a [checkout plugin](https://github.com/commercelayer/commercelayer-cli-plugin-checkout/blob/main/README.md) that can generate the URL from an [SKU](https://commercelayer.io/docs/data-model/skus), [Bundle](https://commercelayer.io/docs/data-model/bundles), or an order ID using the command below:

```
commercelayer checkout -S <skuCode>:<quantity> --open
```

You can use the `-S` flag for SKUs, `-B` flag for bundles, and `-O` flag for order ID.